dataactcore/config_example.yml

```diff
@@ -68,6 +68,10 @@ broker:
     # AWS credentials for uploading files. Ignored if use_aws is false.
     aws_create_temp_credentials: true
 
+    # S3 filenames for SF-133 file, only required if planning to load SF-133 table
+    sf_133_folder: config
+    sf_133_file_one: sf_133_part1.csv
+    sf_133_file_two: sf_133_part2.csv
```

Comment:
Apologies for not providing any context re: the two SF-133 files. They sent two .xls files b/c the data exceeded the row limit. Now that they're in .csv form, should we just combine them into a single file for simplicity?

This method of passing around SF-133 data is temporary anyway, so I don't anticipate that we'd have to regularly combine two files to get the entire dataset.